yaml/water.yaml

```diff
@@ -18,8 +18,8 @@ globals:
   - &water_standard_min_zoom
     case:
       - when: { way_area: { min: 102400 } }
-        then: 9
-      - else: { clamp: { min: 9, max: 16, value: { sum: [ { col: zoom }, 1.066 ] } } }
+        then: 8
```

Comment:
Where the JINJA prevents OSM water from being included (regardless of their min_zoom) until zoom 8.